modules/backups.nix

```diff
@@ -0,0 +1,76 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.backups;
+  duplicity-filelist = pkgs.writeText "filelist.txt" ''
```

Comment:
I think it would be good if the filelist can be extended with extra lines.

Generally, the most bare-bones but still helpful version backup module is just the list of files that are important to back up. That would  then allow users to integrate nix-bitcoin with their own backup tools without having to go through the step of having the `localBackups` directory (and an additional duplicity password). Therefore it would be nice if (1) the filelist can read from other modules and (2) the filelist can be read from the operator user's shell.

Reply:
> I think it would be good if the filelist can be extended with extra lines.
Resolved

Reply:
> Generally, the most bare-bones but still helpful version backup module is just the list of files that are important to back up. That would then allow users to integrate nix-bitcoin with their own backup tools without having to go through the step of having the localBackups directory (and an additional duplicity password). Therefore it would be nice if (1) the filelist can read from other modules and (2) the filelist can be read from the operator user's shell.

The operator user can already read the filelist from the nix-bitcoin node's nix store.

We could make the filelist an option that defaults to our filelist, similar to nodeinfo script, so it can be read from all modules. I don't think it should be too difficult to translate the duplicity filelist syntax into something other programs understand.